backend/src/contexts/users/actions/CreateUserAction.ts

```diff
@@ -15,6 +16,7 @@ export class CreateUserAction {
 
     await this.checkUsernameIsNotUsed(username);
     await this.checkEmailIsNotUsed(email);
+    await this.checkPasswordIsNotShort(password);
```

Comment:
This validation It's better to put in the UserEntity because it's a invariant from the entity not from the repository how are they if there is already a user with this username or email.

Reply:
Done